cpp/lib/src/ZoteroHarvesterDownload.cc

```diff
@@ -345,7 +345,14 @@ bool Crawler::getNextPage(CrawlResult * const crawl_result) {
                                            | WebUtil::ATTEMPT_TO_EXTRACT_JAVASCRIPT_URLS);
 
     std::vector<WebUtil::UrlAndAnchorTexts> urls_and_anchor_texts;
-    WebUtil::ExtractURLs(download_result.response_body_, next_url, WebUtil::ABSOLUTE_URLS, &urls_and_anchor_texts, EXTRACT_URL_FLAGS);
+
+    // Some pages are evil and include null bytes in strange places...
```

Comment:
Would it be better to put this inside ExtractURLs?

Reply:
Maybe, but I thought due to performance it reasons should execute as seldom as possible.